client/db/bolt/db.go

```diff
@@ -298,9 +299,30 @@ func (db *BoltDB) CreateAccount(ai *dexdb.AccountInfo) error {
 	})
 }
 
+// Make account info key from client pub key
+func accountInfoKey(accountID account.AccountID) []byte {
+	acctKey := make([]byte, len(accountID))
+	copy(acctKey, accountID.String())
+	return acctKey
+}
```

Comment:
You can remove `accountInfoKey` since you can get slice from an array via `[:]`.  And I don't think we want the key to actually be based on the string hexadecimal encoding of the `AccountID` when the `AccountID` bytes themselves will do fine (and is twice as space efficient and much more efficient without the conversions).